### PR TITLE
Fix Bug 1185468, set country as required and style as other required fields when state is invalid

### PR DESCRIPTION
**Pull request by schalkneethling**
@alexgibson r?

### CODE REVIEW

File: bedrock/mozorg/forms.py [hunk 1]
```diff
@@ -192,7 +192,7 @@ def __init__(self, locale, *args, **kwargs):
         regions.insert(0, self.empty_choice)
         super(ContributeSignupForm, self).__init__(*args, **kwargs)
         self.locale = locale
-        self.fields['country'] = forms.ChoiceField(choices=regions, widget=L10nSelect)
+        self.fields['country'] = forms.ChoiceField(choices=regions, widget=L10nSelect(attrs={'required': 'required'}))
```
Comment (alexgibson) [inline bedrock/mozorg/forms.py]: Nit: this line is pretty long, might be easier to read if indented?

File: media/css/mozorg/contribute/contribute-2015.less [hunk 2]
```diff
@@ -154,6 +154,12 @@ textarea:focus {
     }
 }
 
+.select.invalid {
+    border-color: #a91300;
+    @shadow: 0 0 0 2px rgba(255,80,80,0.4);
```
Comment (alexgibson) [inline media/css/mozorg/contribute/contribute-2015.less]: Any need for a variable here? Could just use `.box-shadow(0 0 0 2px rgba(255, 80, 80, 0.4));`?

Comment (alexgibson) [inline media/css/mozorg/contribute/contribute-2015.less]: I guess it might be more readable? Happy to leave as-is.

**Comment by alexgibson** [comment]
r+wc :banana:

File: media/css/mozorg/contribute/contribute-2015.less (hunk 2 above)
Comment (schalkneethling) [inline media/css/mozorg/contribute/contribute-2015.less]: Yeah, copied it from sandstone-resp.less ;)

File: bedrock/mozorg/forms.py (hunk 1 above)
Comment (schalkneethling) [inline bedrock/mozorg/forms.py]: Agreed

**Comment by schalkneethling** [comment]
@alexgibson updated, thanks!

**Pull request merged**